shell/key-bindings.fish

```diff
@@ -1,7 +1,9 @@
 # Key bindings
 # ------------
 function fzf_key_bindings
-
+  set -g __FZF_FISH_MAJOR (echo $FISH_VERSION | cut -f1 -d.)
```

Comment:
According to Wikipedia, fish is an "exotic" shell!

Fish does have string manipulation functions, but only as of 2.3 (see https://fishshell.com/docs/current/commands.html#string ). I figured supporting fish <2.3 was more important saving subprocess invocations.